utils/src/test/kotlin/CopyrightStatementsProcessorTest.kt

```diff
@@ -34,11 +36,19 @@ class CopyrightStatementsProcessorTest : WordSpec() {
             "return a result with items merged by owner and prefix, sorted by owner and year" {
                 val input = File("src/test/assets/copyright-statements.txt").readLines()
 
-                val result = yamlMapper.writeValueAsString(processor.process(input))
+                val result = processor.process(input).toYaml()
```

Comment:
Commit message: Isn't this more than just extracting to a helper function (without changing behavior), as you also call `setVisibility` below?

Reply:
Sorry, some mistake happened during rebase. Done.